src/20250826_Multi_SeptemberFundingUpdate/AaveV3Ethereum_SeptemberFundingUpdate_20250826.sol

```diff
@@ -0,0 +1,142 @@
+import {IWETH} from 'aave-v3-origin/contracts/helpers/interfaces/IWETH.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/src/interfaces/IProposalGenericExecutor.sol';
+
+/**
+ * @title September Funding Update
+ * @author @TokenLogic
+ * - Snapshot: Direct-to-AIP
+ * - Discussion: https://governance.aave.com/t/direct-to-aip-september-2025-funding-update/22915
+ */
+contract AaveV3Ethereum_SeptemberFundingUpdate_20250826 is IProposalGenericExecutor {
+  using CollectorUtils for ICollector;
+
+  /// AAVE Buybacks allowance
+  uint256 public constant AFC_USDC_ALLOWANCE = 2_000_000e6;
+  uint256 public constant AFC_USDT_ALLOWANCE = 2_000_000e6;
+
+  /// ALC CRV allowance
+  uint256 public constant ALC_CRV_ALLOWANCE = 30_000 ether;
+
+  /// MERIT GHO allowance
+  uint256 public constant MERIT_GHO_ALLOWANCE = 3_000_000 ether;
+
+  /// https://etherscan.io/address/0x060373D064d0168931dE2AB8DDA7410923d06E88
+  address public constant MILKMAN = 0x060373D064d0168931dE2AB8DDA7410923d06E88;
+
+  /// https://etherscan.io/address/0xe80a1C615F75AFF7Ed8F08c9F21f9d00982D666c
+  address public constant PRICE_CHECKER = 0xe80a1C615F75AFF7Ed8F08c9F21f9d00982D666c;
+
+  /// https://etherscan.io/address/0x455e53CBB86018Ac2B8092FdCd39d8444aFFC3F6
+  address public constant POL = 0x455e53CBB86018Ac2B8092FdCd39d8444aFFC3F6;
+
+  /// https://etherscan.io/address/0xc980508cC8866f726040Da1C0C61f682e74aBc39
+  address public constant PLASMA_BRIDGE = 0xc980508cC8866f726040Da1C0C61f682e74aBc39;
+
+  /// Reimbursements
+  address public constant ACI = 0xac140648435d03f784879cd789130F22Ef588Fcd;
+  uint256 public constant ACI_REIMBURSEMENT = 0.2415 ether;
+
+  address public constant TOKEN_LOGIC = 0xAA088dfF3dcF619664094945028d44E779F19894;
+  uint256 public constant TL_REIMBURSEMENT = 9_000 ether;
+
+  uint256 public constant ETH_TO_DEPOSIT = 223 ether;
+
+  function execute() external {
+    _approvals();
+    _swaps();
+    _reimbursements();
+  }
+
+  function _approvals() internal {
+    // AFC aEthUSDC Approval
+    AaveV3Ethereum.COLLECTOR.approve(
+      IERC20(AaveV3EthereumAssets.USDC_A_TOKEN),
+      MiscEthereum.AFC_SAFE,
+      AFC_USDC_ALLOWANCE
+    );
+
+    // AFC aEthUSDT Approval
+    AaveV3Ethereum.COLLECTOR.approve(
+      IERC20(AaveV3EthereumAssets.USDT_A_TOKEN),
+      MiscEthereum.AFC_SAFE,
+      AFC_USDT_ALLOWANCE
+    );
+
+    // ALC aEthCRV Approval
+    AaveV3Ethereum.COLLECTOR.approve(
+      IERC20(AaveV3EthereumAssets.CRV_A_TOKEN),
+      MiscEthereum.ALC_SAFE,
+      ALC_CRV_ALLOWANCE
+    );
+
+    // MERIT aEthLidoGHO Approval
+    AaveV3EthereumLido.COLLECTOR.approve(
+      IERC20(AaveV3EthereumLidoAssets.GHO_A_TOKEN),
+      MiscEthereum.MERIT_AHAB_SAFE,
+      MERIT_GHO_ALLOWANCE
+    );
+  }
+
+  function _swaps() internal {
+    // ETH
+    AaveV3Ethereum.COLLECTOR.transfer(
+      IERC20(AaveV3Ethereum.COLLECTOR.ETH_MOCK_ADDRESS()),
+      address(this),
+      ETH_TO_DEPOSIT
```

Comment:
i think fine, but perhaps in the future better use .balance instead of fixed amount.